android-json-form-wizard/src/main/java/com/vijay/jsonwizard/constants/JsonFormConstants.java

```diff
@@ -167,6 +167,15 @@
     public static final String OPTION_KEY = "option_key";
     public static final String PARENT_KEY = "parent_key";
     public static final String IS_NUMBER_SELECTOR_MENU = "is_number_selector_menu";
+    public static final String COUNTDOWN_TIMER = "countdown_timer";
+    public static final String COUNTDOWN_TIME_UNIT = "countdown_time_unit";
+    public static final String COUNTDOWN_TIME_VALUE = "countdown_time_value";
+    public static final String COUNTDOWN_INTERVAL = "countdown_interval";
```

Comment:
Could you rename this to `COUNTDOWN_INTERVAL_SECS` for clarity?

Reply:
Also validate that this value is less than the `COUNTDOWN_TIME_VALUE` and have a default fallback.

Reply:
Will have a default of 1-second interval since that's the least interval duration we should allow